### DISCUSSION TITLE
Welcome to mentor-program-4th Discussions!

### DISCUSSION THREAD

**Discussion by aszx87410** [Lidemy/mentor-program-4th#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ishin4554** [comment]
test

**Comment by ishin4554** [comment]
testtest

**Comment by ishin4554** [comment]
testtest

**Comment by keronscribe** [comment]
hello beta🔓
